### DISCUSSION TITLE
RISC-V executable running on x86-64 via WoW64

### DISCUSSION THREAD

**Discussion by Torinde** [rizwan3d/SharpRISCV#23] [General]
If a RISC-V edition of Windows is made probably it'll have a WoW64 JIT DLL for running x86/x86-64 programs.

What about the reverse scenario? Can your RISC-V executables be run via another WoW64 JIT DLL on current Windows PCs?

**Comment by rizwan3d** [comment]
Are you asking about running RISC-V exe on windows x86/x86-64?

**Reply by Torinde** [reply]
Yes.

**Reply by rizwan3d** [reply]
A project that might be relevant to this scenario is [riscv64-vm](https://github.com/rizwan3d/riscv64-vm), which is designed to emulate RISC-V architecture on platforms like x86-64, allowing you to run RISC-V executables in a virtualized environment on your existing hardware.
